### ISSUE TITLE
peer not authenticated

### ISSUE THREAD

**Issue by DanielRuf** [sissbruecker/gog-java-api#3]
funktioniert irgendwie nur eine bestimmte Zeit lang und dann kommt eine Exception

Ausgelöst wird sie bei

```
try {
            response = client.execute(post, context);
        } catch (IOException e) {
            throw new GogApiException("Error executing POST request. url=" + url, e);
        }
```

in HttpClientUtil.java

**Comment by sissbruecker** [comment]
I'm going to look into that when I find some time.

**Comment by DanielRuf** [comment]
Ok =)

**Comment by sissbruecker** [comment]
@DanielRuf,

I tried to reproduce, but you did not really provide a lot of details to pinpoint the problem. From the looks of it you're having problems with a POST request, of which the login is the only one. Also it seems related to SSH and certificates. However I can execute the _GogApiTest.java_ as many times as I want, it always succeeds, on Win and OSX. I did not import any certificates or did any other fancy stuff to get it working :/

Can you show me your code and a complete stacktrace (post it in a Gist)? Also can you clarify in how it works for "some time"? Do you login multiple times with the same API instance? Do you create multiple instances and call login on each of them?

Also it would be nice (and polite) to keep this discussion in english so other users can benefit from it too :)

**Comment by DanielRuf** [comment]
I just ran the program multiple times and I also printed a list of the certificates to the console, which are supported:

It may be an issue but with the issuer (digicert). Trying to access the website using SSL using Java fails because this is not a trusted root certificate which is also not included in the JDK/JRE (I checked here and these are the only certificates I get in my console) =( 

These are the only CA which are trusted now: 
Issuer: CN=thawte Primary Root CA, OU="(c) 2006 thawte, Inc. - For authorized 
Issuer: CN=UTN - DATACorp SGC, OU=http://www.usertrust.com, O=The USERTRUST Ne 
Issuer: CN=Baltimore CyberTrust Code Signing Root, OU=CyberTrust, O=Baltimore, 
Issuer: EMAILADDRESS=info@valicert.com, CN=http://www.valicert.com/, OU=ValiCe 
Issuer: CN=thawte Primary Root CA - G3, OU="(c) 2008 thawte, Inc. - For author 

This and some other problems with DigiCert might be the cause: http://www.h-online.com/security/news/item/Certified-online-banking-trojan-in-the-wild-1808898.html

I don't know if this is the problem. I changed nothing here, just using JDK 7 U17 / JRE 1.7 U17, which one are you using?

The stacktrace just points to the SSL exception caused by the POST request.
